assets/src/edit-story/panels/layer.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import { Panel, PanelTitle, PanelContent } from './panel';
+
+function LayerPanel() {
+	return (
+		<Panel initialHeight={ 240 }>
```

Comment:
Feel like this height should be defined somewhere better than this...

Reply:
For this PR it'll stay like this. I'll make sure to revisit it in the coming PR implementing the guts of the layer panel (if I forget, I'll submit to public flogging).

Reply:
Why not just move it to `constants` in the root?

Reply:
It will be revisited soon. I don't see a need for premature optimization here.

Reply:
Why is that premature?

Reply:
Because I'm fixing it in the very next PR to something, but the fix includes variables that have no context in this PR.